package.json

```diff
@@ -3,36 +3,39 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@sentry/react": "^5.24.2",
-    "@sentry/tracing": "^5.24.2",
-    "@types/file-saver": "^2.0.1",
-    "@types/jest": "24.0.22",
-    "@types/lodash": "^4.14.116",
-    "@types/node": "12.12.7",
-    "@types/react": "^16.8.4",
-    "@types/react-beautiful-dnd": "^11.0.3",
-    "@types/react-dom": "^16.8.2",
-    "@types/react-helmet": "^6.1.1",
-    "@types/react-router-dom": "^5.1.2",
-    "file-saver": "^2.0.0-rc.3",
-    "firebase": "^7.3.0",
+    "@artsy/fresnel": "^7.1.4",
+    "@sentry/react": "^8.8.0",
+    "@sentry/tracing": "^7.114.0",
+    "@types/file-saver": "^2.0.7",
+    "@types/jest": "29.5.12",
+    "@types/lodash": "^4.17.5",
+    "@types/node": "20.14.2",
+    "@types/react": "^18.3.3",
+    "@types/react-beautiful-dnd": "^13.1.8",
+    "@types/react-dom": "^18.3.0",
+    "@types/react-helmet": "^6.1.11",
+    "@types/react-router-dom": "^5.3.3",
+    "file-saver": "^2.0.5",
+    "firebase": "^10.12.2",
     "lodash": "^4.17.11",
-    "react": "16.11.0",
-    "react-beautiful-dnd": "^12.0.0",
-    "react-dom": "16.11.0",
-    "react-firebase-hooks": "^2.1.0",
-    "react-ga": "^2.5.3",
-    "react-helmet": "^6.1.0",
```

Comment:
`react-helmet@6.1.0` did actually work on my machine

```diff
package.json --- JSON
23 23     "react-dom": "18.3.1",
24 24     "react-firebase-hooks": "^5.1.1",
25 25     "react-ga": "^3.3.1",
.. 26     "react-helmet": "^6.1.0",
26 27     "react-router-dom": "^5.3.4",
27 28     "react-scripts": "5.0.1",
28 29     "semantic-ui-css": "^2.5.0",

src/pages/Committee.tsx --- 1/2 --- TypeScript TSX
40 40 import {logClickSetupCommittee} from '../modules/analytics';
41 41 import {CommitteeData, CommitteeID, DEFAULT_COMMITTEE} from "../models/committee";
42 42 import { createMedia } from '@artsy/fresnel';
.. 43 import { Helmet } from 'react-helmet';
43 44
44 45 interface DesktopContainerProps {
45 46   menu?: React.ReactNode;

src/pages/Committee.tsx --- 2/2 --- TypeScript TSX
362 363
363 364     return (
364 365       <Container text style={{ padding: '1em 0em' }}>
... 366         <Helmet>
365 367           <title>{`${committee?.name} - Muncoordinated`}</title>
... 368         </Helmet>
366 369         <Header as="h1">
367 370           <Input
368 371             value={committee ? committee.name : ''}
```
<img width="239" alt="image" src="https://github.com/MaxwellBo/Muncoordinated-2/assets/5368490/0800a981-fdc5-4752-9ccc-eb7af6283ea6">
not sure what errors you were getting.

Otherwise we'll need the React 19 release candidate (`yarn add --exact react@rc react-dom@rc`)

Reply:
Have added it back in, seems alright so far